docs/advanced/extensibleenums.md

```diff
@@ -7,216 +7,139 @@ Extensible Enums are an enhancement of specific Vanilla enums to allow new entri
 All enums that can have new entries implement the `IExtensibleEnum` interface. This interface acts as a marker to allow the `RuntimeEnumExtender` launch plugin service to know what enums should be transformed.
 
 :::warning
-You should **not** be implementing this interface on your own enums. Use maps or registries instead depending on your usecase.
+You should **not** be implementing this interface on your own enums. Use maps or registries instead depending on your usecase.  
+Enums which are not patched to implement the interface cannot have the interface added to them via mixins or coremods due to the order the transformers run in.
 :::
 
 ### Creating an Enum Entry
 
-To create a new enum entry, the static `create` method should be called. This static method is added to all extensible enums and is transformed at runtime to allow new enums to be added.
-
-:::note
-The `create` method should be called within a static final field if you need to refer to the value or within the main mod constructor if you do not.
-:::
-
-The `create` method contains a name parameter followed by the parameters of a supporting constructor. The name parameter represents the name of the enum constant, which is returned by `#name`. The name of the enum constant should be prefixed with your mod id followed by an underscore (`_`) to avoid conflicts between mods that decide to add the same enum constant (e.g., adding a enum constant called `test` with mod id `examplemod` should be `EXAMPLEMOD_TEST`).
-
-:::note
-Since extensible enums are added at runtime, technically, any UTF-8 character can be used for the name parameter. However, it is recommended to only use valid Java identifiers.
-:::
-
-```java
-// In your main mod class
-public static final FireworkExplosion.Shape DUMMY =
-        FireworkExplosion.Shape.create("EXAMPLEMOD_DUMMY", 294, "examplemod_dummy");
+To create new enum entries, a JSON file needs to be created and referenced in the `neoforge.mods.toml` with the `enumExtensions` entry of a `[[mods]]` block. The specified path must be relative to the `resources` directory:
+```toml
+# In neoforge.mods.toml:
+[[mods]]
+## The file is relative to the output directory of the resources, or the root path inside the jar when compiled
+## The 'resources' directory represents the root output directory of the resources
+enumExtensions="META-INF/enumextensions.json"
```

Comment:
I think this also needs to be added to modfiles.md as a new option as well. That can probably link to this page.